src/cogs/auto_mod.py

```diff
@@ -94,95 +92,32 @@ async def on_message(self, msg):
                 + f"Flagged content: ```\n{duplicate_message.clean_content}```"
             )
 
-    @commands.group(hidden=True, aliases=["duplication"])
+    @commands.group(aliases=["duplication"])
     @commands.guild_only()
+    @commands.has_role(conf.staff_role)
```

Comment:
doesn't has_role take the role name?

Reply:
https://github.com/Rapptz/discord.py/blob/async/discord/ext/commands/core.py#L779

Reply:
That's an old branch, the current version of dpy does support it:
> If an integer is specified, you must give the exact snowflake ID of the role.

[Source](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.has_role)